### PR TITLE
Update blocks.json to 1.12

### PR DESCRIPTION
**Pull request by boy0001**
used https://github.com/wizjany/ForgeUtils

### CODE REVIEW

File: worldedit-core/src/main/resources/com/sk89q/worldedit/world/registry/blocks.json [hunk 1]
```diff
@@ -1700,7 +1700,7 @@
       "slipperiness": 0.6,
       "renderedAsNormalBlock": false,
       "liquid": false,
-      "solid": true,
+      "solid": false,
```
Comment (Minecrell) [inline worldedit-core/src/main/resources/com/sk89q/worldedit/world/registry/blocks.json]: This looks wrong, cobweb is solid in the Minecraft server

Comment (boy0001) [inline worldedit-core/src/main/resources/com/sk89q/worldedit/world/registry/blocks.json]: It appears to have changed in 1.12

Comment (kashike) [inline worldedit-core/src/main/resources/com/sk89q/worldedit/world/registry/blocks.json]: No, it hasn't.

Comment (Minecrell) [inline worldedit-core/src/main/resources/com/sk89q/worldedit/world/registry/blocks.json]: Yes, it hasn't changed. I've generated the file myself and this was still true. How did you generate it? You may have used the wrong method depending on your mappings.

**Pull request closed without merging**